### PR TITLE
Enum test

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add test to check that the enumerations defined in the various xml files under https://github.com/openmicroscopy/openmicroscopy/tree/develop/components/model/resources/mappings
are correctly inserted during the installation process

The test checks that the values read from the files match the ones in the DB.
It uses method from `ITypes`
# Testing this PR

check https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-java/lastCompletedBuild/testngreports/integration/ConfiguratuionServiceTest/testSupportedEnumerations/
# Related reading

Related to https://github.com/openmicroscopy/openmicroscopy/pull/4820

cc @mtbc

### CODE REVIEW

File: components/tools/OmeroJava/test/integration/ConfigurationServiceTest.java [hunk 1]
```diff
@@ -203,4 +234,191 @@ public void testSupportedFormats() throws Exception {
...
+
+    /**
+     * Returns the enumeration value.
+     *
+     * @param object The enumeration object.
+     * @return See above.
+     */
+    private String getEnumValue(IObject object)
+    {
+        RString value = null;
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/ConfigurationServiceTest.java]: For convenience another option might be some ugly trick with `PropertyUtils.getNestedProperty(object, "value.value")`.

Comment (jburel) [reply components/tools/OmeroJava/test/integration/ConfigurationServiceTest.java]: good point, I extended some existing code from insight.
Let me try that

Comment (jburel) [reply components/tools/OmeroJava/test/integration/ConfigurationServiceTest.java]: better that way so we do not have to modify the test when a new enum is added

**Comment by mtbc** [comment]
Good to merge.

**Pull request merged**